packages/editor/src/components/parts/program/configuration/ConfigurationPart.tsx

```diff
@@ -25,23 +25,12 @@ const ConfigurationPart = () => {
   const { context } = useEditorContext();
   const editorItems = useMeta('meta/program/editor', { context, type: config.javaClass }, []).data;
 
-  function isLabel(object: Widget): object is Label {
-    return 'text' in object;
-  }
-
-  function isScript(object: Widget): object is Script {
-    return 'requiredType' in object;
-  }
-
-  function isText(object: Widget): object is Text {
-    return !isLabel(object) && !isScript(object);
-  }
-
   const renderWidgetComponent = (widget: Widget) => {
-    if (isLabel(widget)) {
-      const message = widget.text;
+    if (widget.widgetType === 'LABEL') {

```

Comment:
why remove this typegard?
just check in the typegard if the widget.widgetType is label or whatever, but then you don't need to manually cast it to a label